lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,7 +59,16 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
-
+    if not isinstance(word_frequencies, dict):
+        return None
+    pairs_dictionary = {}
+    for token in word_frequencies:
+        for i, index in enumerate(token):
+            token_pair = (token[index], token[index + 1])
```

Comment:
Do you ever encounter any errors with this line? Such as Value error or Index error...

Reply:
I think that I had some errors with this line but then they disappeared. I changed it and now it should work properly